cmd/vm-live-migrate-detector/README.md

```diff
@@ -0,0 +1,54 @@
+# vm-live-migrate-detector
```

Comment:
While the command is useful for detecting non-migratable VMs for now, can we rename the utility to something like upgrade-helper and put vm-live-migrate-detector as a sub-command for future extendibility? I'm thinking eventually we want to move all the upgrade_node.go logic into the command to save our time.

Reply:
I separated the `vm-live-migrate-detector` from the root command so that other subcommands could be added to the `upgrade-helper` more easily in the future. Thanks for the suggestion!